.github/workflows/compilemmd.yml

```
@@ -0,0 +1,42 @@
+name: Node.js CI
+
+on:
+  push:
+    paths:
+      - "**/*.mermaid"
+      - "**/*.mmd"
+      - "**/*.md"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
```

[want] この辺にキャッシュも入れたいかも 👀 
```suggestion
      - name: Get yarn cache directory path
        id: yarn-cache-dir-path
        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT

      - name: Run actions/cache@v3
        uses: actions/cache@v3
        if: steps.cache.outputs.cache-hit != 'true'
        id: yarn-cache
        with:
          path: .cache/yarn
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-yarn-

      - name: Check for Yarn and Install if Needed
        run: yarn -v || npm install -g yarn

      - name: Install dependencies
        run: yarn install
```